themes/bene_child/images/svgs/facebook-f.svg

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
why are we adding svgs to the child theme? Do we end up needing them in every child theme we copy for some specific reason?